src/types.js

```diff
@@ -44,17 +44,17 @@ export const types = {
     container: true,
     defaultProps: {
       align: 'center',
-      justify: 'center',
+      justify: 'stretch',
       pad: 'small',
     },
     properties: [
       {
         label: 'Content layout',
         properties: {
-          align: ['stretch', 'start', 'center', 'end'],
+          align: ['stretch', 'start', 'center', 'end', 'baseline'],
           direction: ['column', 'row', 'row-responsive'],
           gap: ['xsmall', 'small', 'medium', 'large', 'xlarge'],
-          justify: ['around', 'between', 'start', 'center', 'end', 'evenly'],
+          justify: ['around', 'between', 'center', 'end', 'evenly', 'start', 'stretch'],
```

Comment:
The list was attempting to be ordered more for a combination of frequency and correlation. Something feels a bit awkward having `start` come after `end`. One could argue that the order should be: stretch, start, between, around, evenly, center, end.